pkg/login/social/social.go

```diff
@@ -309,7 +310,7 @@ func newSocialBase(name string,
 		autoAssignOrgRole:       autoAssignOrgRole,
 		roleAttributePath:       info.RoleAttributePath,
 		roleAttributeStrict:     info.RoleAttributeStrict,
-		skipOrgRoleSync:         skipOrgRoleSync,
+		skipOrgRoleSyncBase:     skipOrgRoleSync,
```

Comment:
Why do we need `skipOrgRoleSyncBase` instead of `skipOrgRoleSync`?

Reply:
We do not really need it right now. Thought I'd rename to have to separate fields that can be set from the configuration. One in the overall scope for Auth (the one we want to deprecate) and one that is set for the specific OAuth integration.

Does this make sense?

Reply:
We already have the option to specify `skip_org_role_sync` under different sections in the configuration. I also don't really see why this change is necessary.